boards/DIM/Inc/App/states/App_DriveState.h

```diff
@@ -1,9 +1,36 @@
 #pragma once
 
 #include "App_SharedStateMachine.h"
+#include "App_SharedMacros.h"
+#include "App_SevenSegDisplays.h"
+#include "App_SharedExitCode.h"
+
+typedef enum
+{
+    DCM_LED,
+    DIM_LED,
+    FSM_LED,
+    PDM_LED,
+    BMS_LED,
+    NUM_BOARD_LEDS,
+} BoardLeds;
```

Comment:
Nit: Can you move this back to the `.c`, since I don't think any other file needs access to this.
Generally its better to put the least amount of code in the `.h`.